### PR TITLE
#168000401 fixing create article mocking tests

### PR DESCRIPTION
**Pull request by habinezadalvan**
#### What does this PR do?

- It only mocks the function that sends an image on cloudinary instead of the whole route.

#### Description of Task to be completed?

- Added a ternary operator that sends string in case of the test environment. 

- removed the fake route

#### How should this be manually tested?
- Clone the repo

- create .env and include all environment variables

- npm install

- git checkout bug-fix-create-article-mocking-test-issue-16800041

- npm run test

#### Any background context you want to provide?

- None

#### What are the relevant pivotal tracker stories?
[168000401](https://www.pivotaltracker.com/story/show/168000401)

#### Screenshots (if appropriate)
Screen Shot 2019-08-20 at 18 00 23

#### Questions:

none

### CODE REVIEW

File: src/controllers/articles.controller.js [hunk 1]
```diff
@@ -25,17 +26,11 @@ class Articles {
...
-        const { secure_url } = await cloudinary.v2.uploader.upload(file.path);
-        return secure_url;
-      })
-    );
+    const images = await cloudinaryHelper(req.files);
 
     if (findUser) {
       const { title } = req.body;
-      const tags = req.body.taglist.split(' ');
+      const tags = req.body.taglist ? req.body.taglist.split(' ') : [];
```
Comment (Mnickii) [inline src/controllers/articles.controller.js]: Is this part of the taglists you're removing?

Comment (habinezadalvan) [reply src/controllers/articles.controller.js]: Yes, that line should be removed also.

Comment (habinezadalvan) [reply src/controllers/articles.controller.js]: I worked on this feedback.

**Review by minega25 — APPROVED** [review]

**Review by Mnickii — COMMENTED** [review]
Fix failing tests

**Comment by habinezadalvan** [comment]
> Fix failing tests

I'm working on it

**Review by Mnickii — APPROVED** [review]
LGTM

**Review by salviosage — APPROVED** [review]
LGTM

**Review by nkalyesubula — APPROVED** [review]
LGTM

**Pull request merged**